### PR TITLE
Fetch target active from bucketdb.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge @geirst @vekterli PR

### CODE REVIEW

**Review by toregge — APPROVED** [review]

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp [hunk 1]
```diff
@@ -277,7 +279,7 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
         SearchReply::Coverage & coverage = reply->coverage;
         coverage.setActive(numActiveLids);
         //TODO this should be calculated with ClusterState calculator.
-        coverage.setTargetActive(numActiveLids);
+        coverage.setTargetActive(bucketdb.getNumActiveDocs());
```
Comment (vekterli) [inline searchcore/src/vespa/searchcore/proton/matching/matcher.cpp:282]: Can this TODO be removed after these changes?

**Review by vekterli — APPROVED** [review]
:+1:

**Review by geirst — APPROVED** [review]
👍

File: searchcore/src/vespa/searchcore/proton/matching/matcher.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/matching/matcher.cpp:281]: Yes, it can

**Pull request merged**